docs/database/database-spans.md

```diff
@@ -268,6 +302,98 @@ Placeholders in a parameterized query SHOULD not be sanitized. E.g. `where id =
 e.g. from `IN (?, ?, ?, ?)` to `IN (?)`, as this can help with extremely long IN-clauses,
 and can help control cardinality for users who choose to (optionally) add `db.query.text` to their metric attributes.
 
+## Generating a summary of the query text
+
+The `db.query.summary` attribute captures a shortened representation of a query text
+which SHOULD have low-cardinality and SHOULD NOT contain any dynamic or sensitive data.
+
+> [!NOTE]
+> The `db.query.text` attribute is intended to identify individual queries. Even though
+> it is sanitized if captured by default, it could still have high cardinality and
+> might reach hundreds of lines of code.
+>
+> The `db.query.summary` is intended to provide a less granular grouping key that
+> can be used as a span name or a metric attribute in common cases. It SHOULD
+> only contain information that has a significant impact on the query, database,
+> or application performance.
+
+Instrumentations that support query parsing SHOULD generate query summary when
+one is not readily available from other sources.
+
+The summary SHOULD preserve the following parts of query in the order they were provided:
+
+- operations such as SQL SELECT, INSERT, UPDATE, DELETE or other commands
+- operation targets such as collection or database names
+
+Instrumentation MAY include additional details such as specific SQL clauses as long
+as summary remains relatively short and its cardinality remains low comparing to
+the `db.query.text`.
+
+The instrumentation SHOULD parse the query and extract a list of operations and
+corresponding targets from the query. It SHOULD set `db.query.summary` attribute
+to the value formatted in the following way:
+
+```
+{operation1} {target1} {operation2} {target2} {target3} ...
+````
+
+Instrumentations SHOULD capture the values of operations and targets as provided
+by the application without attempting to do any case normalization. If the operation
+and target value is populated on `db.operation.name`, `db.collection.name`, `db.namespace`
+or other attributes, it SHOULD match the value used in the `db.query.summary`.
+
+**Examples**:
+
+- Query that consist of a single operation:
+
+   ```sql
+   SELECT *
+   FROM   wuser_table
+   WHERE  username = ?
+   ```
+
+   the corresponding `db.query.summary` is `SELECT wuser_table`.
+
+- Query that performs multiple operations:
+
+   ```sql
+   INSERT INTO shipping_details
+               (order_id,
+               address)
+   SELECT order_id,
+          address
+   FROM   orders
+   WHERE  order_id = ?
+   ```
+
+   the corresponding `db.query.summary` is `INSERT shipping_details SELECT orders`.
+
+- Query that performs an operation that's applied to multiple collections:
+
+   ```sql
+   SELECT *
+   FROM   songs,
+          artists
+   WHERE  songs.artist_id == artists.id
+   ```
+
+  the corresponding `db.query.summary` is `SELECT songs artists`.
+
```

Comment:
```suggestion

- Query that performs an operation that's applied to multiple collections with double-quotes or other punctuation:

    ```sql
    SELECT *
    FROM   "song list",
           'artists'
    ```

    the corresponding `db.query.summary` is `SELECT "songs list" 'artists'`.

```

Could we include this in examples? So it could guide future readers to deal with this situation.

Related comments: https://github.com/open-telemetry/semantic-conventions/pull/1482#discussion_r1803546355

Reply:
added!